ababot/bot/Cargo.toml

```diff
@@ -12,4 +12,7 @@ dir_macros = { path = "../dir_macros" }
 serenity = { version = "0.11", default-features = false, features = ["client", "builder", "model", "http", "gateway", "client", "cache", "utils", "rustls_backend"] }
 tracing = "0.1"
 tokio = { version = "1", features =  ["full"] }
-rand = { version = "0.8", optional = true }
\ No newline at end of file
+rand = { version = "0.8", optional = true }
+serde = "1.0.147"
+serde_json = "1.0.87"
+yahoo_finance_api = "=1.2.2"
```

Comment:
Bad

Reply:
yahoo needs to stay like that. Other ones fixed